package.json

```diff
@@ -32,7 +32,7 @@
   "dependencies": {
     "canonical-path": "0.0.2",
     "catharsis": "^0.8.1",
-    "change-case": "^2.1.0",
+    "change-case": "3.0.0",
```

Comment:
But in any case the commit message here needs to be a bit clear. E.g.

```
BREAKING CHANGE:
The `nunjucks` package rendering filter `sentenceCase` now capitalizes the first letter of the sentence. If you relied on the previous behaviour then you should use the `noCase` filter instead.
```

And we should also add the `noCase` filter into the list of filters.